### PR TITLE
Log all jobs, not just failed ones

### PR DESCRIPTION
**Pull request by jonmv**
@tokle please merge :)

### CODE REVIEW

**Review by tokle — APPROVED** [review]

File: controller-server/src/main/java/com/yahoo/vespa/hosted/controller/deployment/DeploymentJobExecutor.java [hunk 1]
```diff
@@ -42,6 +42,7 @@ public DeploymentJobExecutor(Controller controller, Duration triggeringInterval,
     protected void maintain() {
         controller().applications().deploymentTrigger().deploymentQueue().takeJobsToRun()
                 .forEach(buildJob -> executor.execute(() -> {
+                    log.log(Level.INFO, "Attempting to trigger " + buildJob + " in Screwdriver.");
```
Comment (tokle) [inline controller-server/src/main/java/com/yahoo/vespa/hosted/controller/deployment/DeploymentJobExecutor.java]: I don't see what is being logged here. The BuildJob has no toString so probably not much...
I think we should include: tenant, application, screwdriverId and job (or zone/region)

**Review by tokle — CHANGES_REQUESTED** [review]
see review comments

File: controller-server/src/main/java/com/yahoo/vespa/hosted/controller/deployment/DeploymentJobExecutor.java (hunk 1 above)
Comment (jonmv) [reply controller-server/src/main/java/com/yahoo/vespa/hosted/controller/deployment/DeploymentJobExecutor.java]: There is a `toString`, but there is a second `BuildJob` class now, which you probably looked at.

Comment (jonmv) [reply controller-server/src/main/java/com/yahoo/vespa/hosted/controller/deployment/DeploymentJobExecutor.java]: I'm rewriting this in any case now, so will print more information. Currently, only project ID and job type are available, and those are included in the `toString`./

Comment (tokle) [reply controller-server/src/main/java/com/yahoo/vespa/hosted/controller/deployment/DeploymentJobExecutor.java]: OK, but this only includes project and name

**Pull request merged**